tests/validation_tests/test_pre_checks.py

```diff
@@ -279,3 +279,28 @@ def test_full_pre_transformation_validation_pipeline_failure_place_vs_fund(valid
             expected_values={"TD"},
         )
     ]
+
+
+def test_full_pre_transformation_validation_pipeline_failure_place_null(valid_workbook_round_four):
+    valid_workbook_round_four["2 - Project Admin"][4] = [
```

Comment:
These tests could also be a nice candidate for parametrisation like:

```
@pytest.fixture
def mocked_project_admin_sheet(valid_workbook_round_four, request):
    valid_workbook_round_four["2 - Project Admin"][4] = [
        "",
        "",
        "",
        "",
        "",
        request.param["fund"],
        request.param["place"],
        "",
        "",
    ]
    return valid_workbook_round_four


@pytest.mark.parametrize("mocked_project_admin_sheet, expected",
                         [({"fund": "Future_High_Street_Fund", "place": "Newark"}, [
                             WrongInputFailure(
                                 value_descriptor="Place Name vs Fund Type",
                                 entered_value="HS",
                                 expected_values={"TD"},
                             )
                         ]),
                          ({"fund": "Future_High_Street_Fund", "place": ""}, [
                              WrongInputFailure(
                                  value_descriptor="Place Name",
                                  entered_value="",
                                  expected_values=TF_PLACE_NAMES_TO_ORGANISATIONS.keys(),
                              )
                          ])], indirect=["mocked_project_admin_sheet"])
def test_full_pre_transformation_validation_pipeline_failure_place_vs_fund(mocked_project_admin_sheet, expected):
    with pytest.raises(ValidationError) as e:
        validate_before_transformation(mocked_project_admin_sheet, 4, ["Newark"])
    assert e.value.validation_failures == expected

```

Reply:
I've slept on pytest.parameterize, used the above and added the null Fund Type case